### PR TITLE
Walnut cup: autocomplete in search (with algolia)

### PR DESCRIPTION
**Pull request by modernserf**
Notes:
- This isn't pixel-accurate to the mockup, but I figure refinements can be done in a subsequent PR.
- After #381 is merged, I will enable collection items.
- In the mockup, team and collection results show the avatars of their constituent members. We can't efficiently store that in Algolia, though. Can we limit those to the search results page?
- a few other important fields that we _can_ store still need to be backfilled, but that can be independent of this PR.

### CODE REVIEW

File: src/components/search/form.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  return (
+    <PopoverContainer>
+      {({ visible, setVisible }) => (
+        <form
+          action="/search"
+          method="get"
+          role="search"
+          onSubmit={onSubmit}
+          autoComplete={algoliaFlag ? 'off' : 'on'}
```
Comment (ttseng) [inline src/components/search/form.js]: is this prop needed?

Comment (modernserf) [reply src/components/search/form.js]: yeah -- this disables the browser's built-in autocomplete

Comment (clottman) [reply src/components/search/form.js]: Some people using iOS ask for auto-capitalize off as well - should we add that in here, too?

**Comment by clottman** [comment]
Left some accessibility notes in Slack. :)

**Pull request merged**